javacord-api/src/main/java/org/javacord/api/entity/message/MessageUpdater.java

```diff
@@ -38,4 +42,39 @@ public CompletableFuture<Message> applyChanges() {
     public CompletableFuture<Message> replaceMessage() {
         return delegate.edit(message, true);
     }
+
+
+    /**
+     * Removes an attachment from the message.
+     *
+     * @param attachment The attachment to remove.
+     */
+    public void removeExistingAttachment(Attachment attachment) {
+        delegate.removeExistingAttachment(attachment);
+    }
```

Comment:
The same for the methods below
```suggestion
    public MessageUpdater removeExistingAttachment(Attachment attachment) {
        delegate.removeExistingAttachment(attachment);
        return this;
    }
```